### PR TITLE
Removed inline style

### PR DESCRIPTION
**Pull request by anthonySchafrik**
Removed inline style
made app look like it was not squished  together
Add more style

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "format:check": "prettier --list-different \"src/**/*.{js,jsx,css,json}\"",
     "lint": "eslint \"**/*.{js,jsx}\" --quiet",
     "build": "parcel build src/index.html --no-source-maps ",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "server-dev": "nodemon server/index.js",
```
Comment (jpdaprato) [inline client/package.json]:
```suggestion
```

Comment (jpdaprato) [reply client/package.json]: Not needed for client; only for server

File: client/src/Callback/Callback.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import goblin from "../images/goblin.png";
+// import goblin from "../images/goblin.png";
```
Comment (jpdaprato) [inline client/src/Callback/Callback.jsx]:
```suggestion
```

Comment (jpdaprato) [reply client/src/Callback/Callback.jsx]: Not needed anymore

File: client/src/Home/Home.jsx [hunk 3]
```diff
@@ -18,25 +22,25 @@ class Home extends Component {
   render() {
     const { isAuthenticated } = this.props.auth;
     return (
-      <div className="container">
+      <Wrapper className="container">
```
Comment (jpdaprato) [inline client/src/Home/Home.jsx]:
```suggestion
      <div className="container">
```

File: client/src/Home/Home.jsx [hunk 4]
```diff
@@ -18,25 +22,25 @@ class Home extends Component {
...
         {isAuthenticated() && (
-          <Wrapper>
+          <div>
```
Comment (jpdaprato) [inline client/src/Home/Home.jsx]:
```suggestion
          <Wrapper>
```

File: client/src/Home/Home.jsx [hunk 5]
```diff
@@ -18,25 +22,25 @@ class Home extends Component {
...
             <LinkItems />
             <Link to="/enter-purchase">
               <button>Enter a Purchase</button>
             </Link>
-          </Wrapper>
+          </div>
```
Comment (jpdaprato) [inline client/src/Home/Home.jsx]:
```suggestion
          </Wrapper>
```

File: client/src/Home/Home.jsx [hunk 6]
```diff
@@ -18,25 +22,25 @@ class Home extends Component {
...
-            <a style={{ cursor: "pointer" }} onClick={this.login.bind(this)}>
+            <CursorPointer onClick={this.login.bind(this)}>
               Log In
-            </a>{" "}
+            </CursorPointer>{" "}
             to continue.
           </h4>
         )}
-      </div>
+      </Wrapper>
```
Comment (jpdaprato) [inline client/src/Home/Home.jsx]:
```suggestion
      </div>
```

File: client/src/components/CashFlowMeter.jsx [hunk 7]
```diff
@@ -1,4 +1,10 @@
 import React from "react";
+import styled from "react-emotion";
+
+//Styled Components
+const StopLightBottonSpacing = styled("div")`
```
Comment (jpdaprato) [inline client/src/components/CashFlowMeter.jsx]:
```suggestion
const StopLight = styled("div")`
```

File: client/src/components/CashFlowMeter.jsx [hunk 8]
```diff
@@ -47,9 +53,9 @@ const CashFlowMeter = props => {
   };
 
   return (
-    <div style={{ marginBottom: "2%" }}>
+    <StopLightBottonSpacing>
```
Comment (jpdaprato) [inline client/src/components/CashFlowMeter.jsx]:
```suggestion
    <StopLight>
```

File: client/src/components/CashFlowMeter.jsx [hunk 9]
```diff
@@ -47,9 +53,9 @@ const CashFlowMeter = props => {
...
       {stopLight(props.currentCashFlowAmount - props.purchaseAmount)}
-    </div>
+    </StopLightBottonSpacing>
```
Comment (jpdaprato) [inline client/src/components/CashFlowMeter.jsx]:
```suggestion
    </StopLight>
```

File: client/src/components/EnterPurchase.jsx [hunk 10]
```diff
@@ -7,11 +7,46 @@ import CashFlowMeter from "./CashFlowMeter.jsx";
...
   text-align: center;
 `;
 
+const CashCreditOption = styled("div")`
+  display: flex;
+  margin: 2%;
+  margin-bottom: 5%;
+`;
+
+const RepeatPurchaseFrequencySelecterDiv = styled("div")`
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
const RepeatPurchaseFrequencySelectorDiv = styled("div")`
```

File: client/src/components/EnterPurchase.jsx [hunk 11]
```diff
@@ -7,11 +7,46 @@ import CashFlowMeter from "./CashFlowMeter.jsx";
...
+  margin: 5%;
+  margin-left: 2%;
+`;
+
+const RepeatPurchaseFrequencySelecterOption = styled("select")`
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
const RepeatPurchaseFrequencySelectorOption = styled("select")`
```

File: client/src/components/EnterPurchase.jsx [hunk 12]
```diff
@@ -7,11 +7,46 @@ import CashFlowMeter from "./CashFlowMeter.jsx";
...
+  height: 40px;
+`;
+
+const CashFlowMeterDiv = styled("div")`
+  margin: 5%;
+  display: flex;
+  flex-direction: row;
+`;
+
+const PayInCashSpacing = styled("input")`
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
const PayInCash = styled("input")`
```

File: client/src/components/EnterPurchase.jsx [hunk 13]
```diff
@@ -29,32 +64,34 @@ class EnterPurchase extends React.Component {
...
+          <h2 style={{ marginRight: "30px" }}>Cash Flow</h2>
           <CashFlowMeter
+            data-test="cash-flow-meter"
             currentCashFlowAmount={currentCashFlowAmount}
             purchaseAmount={purchaseAmount}
           />
-        </div>
-        <div style={{ margin: "2%" }}>
+        </CashFlowMeterDiv>
+        <RepeatPurchaseFrequencySelecterDiv>
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
        <RepeatPurchaseFrequencySelectorDiv>
```

File: client/src/components/EnterPurchase.jsx [hunk 14]
```diff
@@ -29,32 +64,34 @@ class EnterPurchase extends React.Component {
...
           Repeat:
-          <select
+          <RepeatPurchaseFrequencySelecterOption
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
          <RepeatPurchaseFrequencySelectorOption
```

File: client/src/components/EnterPurchase.jsx [hunk 15]
```diff
@@ -29,32 +64,34 @@ class EnterPurchase extends React.Component {
...
             value={purchaseFrequency}
             onBlur={handlePurchaseFrequencyChange}
             onChange={handlePurchaseFrequencyChange}
           >
             <option value="never">Never</option>
             <option value="monthly">Monthly</option>
-          </select>
-        </div>
+          </RepeatPurchaseFrequencySelecterOption>
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
          </RepeatPurchaseFrequencySelectorOption>
```

File: client/src/components/EnterPurchase.jsx [hunk 16]
```diff
@@ -29,32 +64,34 @@ class EnterPurchase extends React.Component {
...
+        </RepeatPurchaseFrequencySelecterDiv>
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
        </RepeatPurchaseFrequencySelectorDiv>
```

File: client/src/components/EnterPurchase.jsx [hunk 17]
```diff
@@ -63,17 +100,17 @@ class EnterPurchase extends React.Component {
             onChange={handlePaymentTypeChange}
           />
           Pay in Cash
-          <input
+          <PayInCashSpacing
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
          <PayInCash
```

File: client/src/components/InputAmount.jsx [hunk 18]
```diff
@@ -13,6 +13,14 @@ const Input = styled("input")`
   width: 55%;
 `;
 
+const PurchaseAmountText = styled("label")`
+  color: red;
+`;
+
+const RedDollorSigh = styled("p")`
```
Comment (jpdaprato) [inline client/src/components/InputAmount.jsx]:
```suggestion
const DollarSign = styled("p")`
```

File: client/src/components/InputAmount.jsx [hunk 19]
```diff
@@ -45,13 +53,13 @@ class InputAmount extends React.Component {
...
-        <label onClick={this.changeLabelToInput}>
+        <PurchaseAmountText onClick={this.changeLabelToInput}>
           ${this.props.purchaseAmount}
-        </label>
+        </PurchaseAmountText>
       );
     } else {
       return (
-        <p>
+        <RedDollorSigh>
```
Comment (jpdaprato) [inline client/src/components/InputAmount.jsx]:
```suggestion
        <DollarSign>
```

File: client/src/components/InputAmount.jsx [hunk 20]
```diff
@@ -62,7 +70,7 @@ class InputAmount extends React.Component {
             /* eslint-disable-next-line */
             autoFocus
           />
-        </p>
+        </RedDollorSigh>
```
Comment (jpdaprato) [inline client/src/components/InputAmount.jsx]:
```suggestion
        </DollarSign>
```

File: client/src/components/SnapshotResults.jsx [hunk 21]
```diff
@@ -142,6 +146,7 @@ class SnapshotResults extends React.Component {
               </button>
 
               <PayDebtOrInvestIt
+                data-test="pay-debt-or-investIt"
```
Comment (jpdaprato) [inline client/src/components/SnapshotResults.jsx]:
```suggestion
                data-test="pay-debt-or-invest-it"
```

Comment (jpdaprato) [reply client/src/components/SnapshotResults.jsx]: Check to see if this needs to be changed elsewhere as well.

Comment (anthonySchafrik) [reply client/src/components/SnapshotResults.jsx]: The - between invest and it is keeping the html kabob style

File: client/src/Home/Home.jsx (hunk 3 above)
Comment (anthonySchafrik) [reply client/src/Home/Home.jsx]: Changing wrapper back to div will null the style and make the connect Accounts and enter purchase button no longer center

**Pull request closed without merging**